### PR TITLE
docs: correct comments in products directory

### LINKED ISSUE #348: Incorrect Documentation in Products Directory
## Description
incorrect documentation of function params in products.service.ts and products.routing.module.ts

## Steps to Reproduce
Go to src/app/products/products.service.ts
Go to src/app/products/products.routing.module.ts

## Expected Behaviour
No params in service file
Product mix resolver and component under wrong comments in routing file

## Actual Behaviour
Says that there are 'any' params
Product mix resolver and component should be under their respective comments

## Settings
Not Applicable

## Screenshots, if any
Not Applicable

### PR DESCRIPTION
**Pull request by jonathanzhang53**
## Description
Reformatted docs in products.service.ts in the comments.

## Related issues and discussion
#348

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/products/products.service.ts [hunk 1]
```diff
@@ -19,23 +19,23 @@ export class ProductsService {
   constructor(private http: HttpClient) { }
 
   /**
-   * @param {any} Fetches recurring deposit products.
+   * Fetches recurring deposit products.
```
Comment (abhaychawla) [inline src/app/products/products.service.ts]: Mention what is being returned in the same line as `@returns` and do not use the word fetches as it is not required but clearly understood from that. See accounting service for reference. As an example:
```
@returns {Observable<any>} Recurring deposit products data.
```

**Comment by jonathanzhang53** [comment]
Ah sorry about the bad formatting for this PR, I made it a while back before I got used to the guidelines. Fixed the issues though.

**Comment by jonathanzhang53** [comment]
I corrected the original issue and this PR and made the changes in the products.routing.module.ts that I specified on the gitter.

File: src/app/products/products-routing.module.ts [hunk 2]
```diff
@@ -13,15 +13,13 @@ import { ProductsComponent } from './products.component';
 import { ManageTaxConfigurationsComponent } from './manage-tax-configurations/manage-tax-configurations.component';
 import { RecurringDepositProductsComponent } from './recurring-deposit-products/recurring-deposit-products.component';
 import { ChargesComponent } from './charges/charges.component';
-import { FixedDepositProductsComponent } from './fixed-deposit-products/fixed-deposit-products.component';
+import { FixedDepositProductsComponent } from './fixed-deposit-products/fixed-deposit-products.component
```
Comment (abhaychawla) [inline src/app/products/products-routing.module.ts]: Missing inverted comma and semicolon, the Travis CI build is failing.

File: src/app/products/products.service.ts [hunk 3]
```diff
@@ -19,32 +19,28 @@ export class ProductsService {
   constructor(private http: HttpClient) { }
 
   /**
-   * @param {any} Fetches recurring deposit products.
-   * @returns {Observable<any>}
+   * @returns {Observable<any>} Recurring deposit products
```
Comment (abhaychawla) [inline src/app/products/products.service.ts]: A similar update might be made for each comment to ensure consistency with `accounting.service.ts`.
```suggestion
   * @returns {Observable<any>} Recurring deposit products data.
```

**Pull request merged**